modules/oadp-usecase-backup-odf.adoc

```diff
@@ -0,0 +1,255 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-use-cases/oadp-usecase-backup-using-odf.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-usecase-backup_{context}"]
+= Backing up an application using {oadp-short} and {odf-short}
+
+In this use case, you back up an application by using {oadp-short} and store the backup in a storage object provided by {odf-first}.
+
+* You create a object bucket claim (OBC) to configure the backup storage location. You use {odf-short} to configure an Amazon S3-compatible object storage bucket. {odf-short} provides MultiCloud Object Gateway (NooBaa MCG) and Ceph Object Gateway, also known as RADOS Gateway (RGW), object storage service. In this use case, you use NooBaa MCG as the backup storage location.
```

Comment:
you use NooBaa MCG  or RGW as the backup storage location.

Reply:
I have mentioned in the intro that users can use both NooBaa MCG and RGW. This use case specifically creates a NooBaa bucket. That is why have pointed that out in the description. Thanks.